packages/monitors/src/ContractMonitor.js

```diff
@@ -96,38 +89,39 @@ class ContractMonitor {
 
     Object.assign(this, createObjectFromDefaultProps(config, defaultConfig));
 
+    // Validate the EMPProps object. This contains a set of important info within it so need to be sure it's structured correctly.
+    const defaultEmpProps = {
+      empProps: {
+        value: {},
+        isValid: x => {
+          // The config must contain the following keys and types:
+          return (
+            Object.keys(x).includes("collateralSymbol") &&
+            typeof x.collateralSymbol === "string" &&
+            Object.keys(x).includes("syntheticSymbol") &&
+            typeof x.syntheticSymbol === "string" &&
+            Object.keys(x).includes("priceIdentifier") &&
+            typeof x.priceIdentifier === "string" &&
+            Object.keys(x).includes("collateralDecimals") &&
+            typeof x.collateralDecimals === "number" &&
+            Object.keys(x).includes("syntheticDecimals") &&
+            typeof x.syntheticDecimals === "number" &&
+            Object.keys(x).includes("priceFeedDecimals") &&
+            typeof x.priceFeedDecimals === "number" &&
+            Object.keys(x).includes("networkId") &&
+            typeof x.networkId === "number"
+          );
+        }
+      }
+    };
+    Object.assign(this, createObjectFromDefaultProps({ empProps }, defaultEmpProps));
+
     // Helper functions from web3.
     this.toWei = this.web3.utils.toWei;
     this.toBN = this.web3.utils.toBN;
     this.utf8ToHex = this.web3.utils.utf8ToHex;
-  }
-
-  // Calculate the collateralization Ratio from the collateral, token amount and token price
-  // This is cr = [collateral / (tokensOutstanding * price)] * 100
-  calculatePositionCRPercent(collateral, tokensOutstanding, tokenPrice) {
```

Comment:
this was not removed, rather, it was put at the bottom of the module like the other `monitor` modules. I also appended an `_` to the front of the function names for consistency.